src/broker/index.js

```diff
@@ -63,25 +73,37 @@ module.exports = class Broker {
 
     // The lock timeout has twice the connectionTimeout because the same timeout is used
     // for the first apiVersions call
-    const lockTimeout = 2 * this.connection.connectionTimeout + this.authenticationTimeout
+    const lockTimeout = 2 * this.connection.connectionTimeout
```

Comment:
This still needs to have the `authenticationTimeout` added, since we're still authenticating within the same flow.